SU2_CFD/src/solvers/CEulerSolver.cpp

```diff
@@ -4311,63 +4309,59 @@ void CEulerSolver::SetFarfield_AoA(CGeometry *geometry, CSolver **solver_contain
 
     /*--- Compute a new value for AoA on the fine mesh only (degrees)---*/
 
-    if (iMesh == MESH_0) AoA = AoA + AoA_inc;
-    else { AoA = config->GetAoA(); }
-
-    /*--- Only the fine mesh stores the updated values for AoA in config ---*/
-
     if (iMesh == MESH_0) {
+      AoA = AoA + AoA_inc;
       config->SetAoA(AoA);
     }
+    else {
+      AoA = config->GetAoA();
+    }
+
+    AoA *= PI_NUMBER/180.0;
```

Comment:
Loose thought: One could make a central "GRAD2RAD = PI_NUMBER/180.0;" as i found 89 instances of this... i mean, by this you already eliminated 5 uses :)

Reply:
Saving some characters does not really solve the issue of forgetting the conversion.